### PR TITLE
Add package magento-mirror-1.9.2.4

### PR DESCRIPTION
**Pull request by diazwatson**
Changes proposed in this pull request:

- Add package magento-mirror-1.9.2.4

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -88,4 +91,4 @@ before_script:
   - source build/travis/before_script.sh
 
 script:
-  - vendor/bin/phpunit --debug --stop-on-error --stop-on-failure
+  - vendor/bin/phpunit --debug --stop-on-error --stop-on-failure
```
Comment (ktomk) [inline .travis.yml]: Looks like a newline is missing here.

File: config.yaml [hunk 2]
```diff
@@ -539,4 +548,4 @@ commands:
       folders:
         - catalog/product/cache
         - css
-        - js
+        - js
```
Comment (ktomk) [inline config.yaml]: Here as well.

**Comment by diazwatson** [comment]
Ups!!
@ktomk should I send a new pull request with those new lines?

**Comment by ktomk** [comment]
@diazwatson: No, no need, I'm just back and will take care of it. You can ignore my comments now. I'll update here.

**Comment by diazwatson** [comment]
cheers :)

**Comment by ktomk** [comment]
This is now part of the develop branch, you can run

    n98-magerun.phar self-update --unstable

to give it a try. I'm just checking it and wondering about the order of the versions, please for yourself as well.

**Comment by diazwatson** [comment]
It works for me but you are right about the order of the versions. 
Can I create another PR to fix it?

**Comment by ktomk** [comment]
@diazwatson: You don't need to, I just fixed it. As you say it's green for you, and it looks good here, I'll release this version. Thanks again for your efforts, taking care and helping testing!

**Comment by ktomk** [comment]
@diazwatson: Version 1.97.15 is out now containing your patch. Thanks again!

**Comment by diazwatson** [comment]
Nice!!
Glad to help :)

**Pull request closed without merging**